style.css

```diff
@@ -16,4 +16,8 @@ header {
 
 main {
   padding: 2em;
+}
+
+h4 {
+	color: Blue;
```

Comment:
Can we use some other color for titles so that they don't look too much like links?